cogs/manga_main.py

```diff
@@ -271,7 +271,11 @@ async def add(self, ctx, manga: Option(str, description="The name of the manga s
         else:
             modeval = "user"
             mode = None
+
+        # error embeds
         setupError = discord.Embed(title="Error", color=0xff4f4f, description="Sorry! Please run the setup command first.")
+        permissionError = discord.Embed(title="Error", color=0xff4f4f, description=("You don't have permission to add manga. Allow roles to add manga with `/server add_role`."))
```

Comment:
I would rather you move this to line 297, just for organization sake as it isn't used anywhere else I believe.